tests/migrations/test_serializer.py

```diff
@@ -0,0 +1,20 @@
+from unittest import TestCase
+from django.db.migrations.serializer import SetSerializer
+
+
+class SetSerializerTest(TestCase):
+    def test_serilize_empty_set(self):
+        orig = set()
+        serializer = SetSerializer(orig)
+        value, imports = serializer.serialize()
+        self.assertEqual(value, 'set()')
+        evaled = eval(value)
+        self.assertEqual(evaled, orig)
+
+    def test_serilizes_to_set_literal(self):
+        orig = {'a'}
+        serializer = SetSerializer(orig)
+        value, imports = serializer.serialize()
+        self.assertEqual(value, "{'a'}")
+        evaled = eval(value)
+        self.assertEqual(evaled, orig)
```

Comment:
Moving migration serializer tests to it's own module could make sense in a separate commit but let's keep these in `test_writer.WriterTests()` for now.